docs/serving/available-steps.md

```diff
@@ -73,26 +73,30 @@ FeaeturesetValidator | Y      | N     | Y      |
  
 
 ## Sources
-| Class name                                       | Description                                   |   
-|--------------------------------------------------|---------------------------------|
-| [mlrun.datastore.BigQuerySource](../api/mlrun.datastore.html#mlrun.datastore.BigQuerySource) | Reads Google BigQuery query results as input source for a flow.  |
-| [mlrun.datastore.CSVSource](https://storey.readthedocs.io/en/latest/api.html#storey.sources.CSVSource) | Reads a CSV file as input source for a flow.   |
-| [storey.sources.DataframeSource](https://storey.readthedocs.io/en/latest/api.html#storey.sources.DataframeSource) | Reads data frame as input source for a flow. | 
-| [mlrun.datastore.HttpSource](../api/mlrun.datastore.html#mlrun.datastore.HttpSource) | Sets the HTTP-endpoint source for the flow. |
-| [mlrun.datastore.KafkaSource](../api/mlrun.datastore.html#mlrun.datastore.KafkaSource) | Sets the kafka source for the flow. |
-| [mlrun.datastore.ParquetSource](https://storey.readthedocs.io/en/latest/api.html#storey.sources.ParquetSource) | Reads the Parquet file/dir as the input source for a flow.  |
-| [mlrun.datastore.StreamSource](../api/mlrun.datastore.html#mlrun.datastore.StreamSource) | Sets the stream source for the flow. If the stream doesn’t exist it creates it.  | 
-
+| Class name                                                                                                        | Description                                                                     | storey | spark | pandas |
+| --------------------------------------------------                                                                | ---------------------------------                                               | ---    | ---   | ---    |
+| [mlrun.datastore.BigQuerySource](../api/mlrun.datastore.html#mlrun.datastore.BigQuerySource)                      | Reads Google BigQuery query results as input source for a flow.                 | N      | Y     | Y      |
+| [mlrun.datastore.SnowFlakeSource]                                                                                 | Reads SnowFlak query results as input source for a flow.                        | N      | Y     | N      |
```

Comment:
Missing a letter, plus apparently the F in Snowflake is not capitalized.
```suggestion
| [mlrun.datastore.SnowFlakeSource]                                                                                 | Reads Snowflake query results as input source for a flow.                        | N      | Y     | N      |
```

Reply:
fixed